channel/npc.go

```diff
@@ -246,24 +246,87 @@ func packetTradeError() mpacket.Packet {
 	return packetNpcShopResult(0xFF)
 }
 
+func storageFlagForInv(inv byte) uint16 {
+	switch inv {
+	case 1:
```

Comment:
Add constants for these magic numbers to align with the handler code in this PR and going forward we should do this as much as possible.